[IMP] web: bs5 utility-class, add font-style

Brieuc-brd opened the pull request:
Define "Bootstrap v5 ready" utility-class to handle font-style.
This class definition can be safely removed after migrating to v5.

This class is needed for https://github.com/odoo/odoo/pull/87639

task-2811202

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/odoo/pull/87637).

Brieuc-brd commented:
Hello @adr-odoo 👋 
This PR is ready for review, could you please take a look at it ? 🙂 

cc. @stefanorigano

adr-odoo commented:
@Brieuc-brd I would prefer that you put this commit in a PR where you use it. 
This allows us to better understand why you need this.

stefanorigano commented:
@Brieuc-brd , it shouldn't be necessary to define `.fst-italic` since we have already `font-italic` that serves the same purpose. To use both before the migration would just complicate the migration process IMO.

@adr-odoo , I think that BRD created a new one 'cause the discuss team prefers to keep PRs organised by topic (see https://github.com/odoo/odoo/pull/87639)

Brieuc-brd commented:
@adr-odoo @stefanorigano 
Based on your review, I have updated the commit 👍 .
I also added, in the description of the commit and the PR, the url of the PR that needs this class to keep track of.

adr-odoo reviewed (approved):
@robodoo r+
